src/RestControllers/AuthorizationController.php

```diff
@@ -925,23 +925,23 @@ public function tokenIntrospection(): void
                     );
                     try {
                         if ($token->verify(new Sha256(), 'file://' . $this->publicKey) === false) {
-                            $result['active'] = 'false';
+                            $result['active'] = false;
```

Comment:
@sjpadgett , I added revoked tests to the automated testing for introspection and noted that in some cases false is being passed as a boolean and others as a string. I changed them to booleans (just guessing since seemed predominant way and how it was documented on the forum), but happy to change them to strings, or just change them back to the way you had (if they were meant to be like that).

Reply:
yep, an oversight on my part as I knew they should be boolean but missed some on my refactor(originally had them as string) nice catch and goes to show testing has its purpose.